tests/reporter/kubernetes.go

```diff
@@ -52,7 +52,6 @@ func (r *KubernetesReporter) SpecSuiteWillBegin(config config.GinkgoConfigType,
 }
 
 func (r *KubernetesReporter) BeforeSuiteDidRun(setupSummary *types.SetupSummary) {
-	r.Cleanup()
```

Comment:
We can check `CI` environment variable here and do this only then ?

Reply:
I would prefer just a sub-directory like `k8s-reporter`.